data/json/construction.json

```diff
@@ -3856,10 +3856,10 @@
     "description": "Build Reinforced Plastic Window",
     "//": "Step 1: Build reinforced plastic window",
     "category": "REINFORCE",
-    "required_skills": [ [ "fabrication", 9 ] ],
+    "required_skills": [ [ "fabrication", 2 ] ],
     "time": "40 m",
     "qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_M", "level": 1 } ] ],
-    "components": [ [ [ "plastic_sheet", 4 ] ], [ [ "nail", 16 ] ], [ [ "2x4", 4 ] ] ],
+    "components": [ [ [ "rigid_plastic_sheet", 4 ] ], [ [ "nail", 16 ] ], [ [ "2x4", 4 ] ] ],
```

Comment:
The description mentions three plastic sheets?

Reply:
huh one of my pushes hasn't pushed

Reply:
"qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_M", "level": 1 } ] ],
    "components": [ [ [ "rigid_plastic_sheet", 2 ] ], [ [ "nail", 16 ] ], [ [ "2x4", 4 ] ] ],